cf-suse-stacks.yml

```diff
@@ -63,23 +73,39 @@ jobs: ##########################################################################
     - get: version
       params: { pre: alpha }
     - get: receipt
+
   - do:
     - task: make-rootfs
       privileged: true
       file: src-ci/tasks/make-rootfs/task.yml
       input_mapping:
         src: stacks-git
+
   - put: artifact
     params:
       file: out/opensuse42-*.tar.gz
       acl: public-read
+
   - put: version
     params: { file: version/version }
+
   - put: receipt
     params:
       file: out/opensuse42-*.receipt.txt
       acl: public-read
+
   - put: receipt-diff
     params:
       file: out/opensuse42-*.receipt.diff
       acl: public-read
+
+  - put: docker-image
+    params:
+      import_file: out/opensuse42.tar.gz
+      skip_download: true
```

Comment:
I believe it doesn't make any difference (but should). Because each "put" has an implied "get": once the resource is uploaded, it is downloaded again right away.

I did copy this from https://github.com/cloudfoundry/buildpacks-ci/blob/8f5560e59b9d87dbd002628da2a539e79f692c3d/pipelines/cflinuxfs2.yml#L669 even though I think it probably doesn't work. Doesn't hurt either though, and maybe will start working one day?